.eslintrc.js

```diff
@@ -3,4 +3,5 @@ module.exports = {
   'ecmaFeatures': {
     'modules': true,
   },
+  'parser': 'babel-eslint',
```

Comment:
sorry for linter stuff commit. I just that I update my `master` before `prop throttle` where merge. 
Close this PR please.